README.md

```diff
@@ -1,8 +1,10 @@
+[![Build Status](https://build.incquerylabs.com/jenkins/buildStatus/icon?job=MagicDraw-OSS%2Fplugin-skeleton%2Fmaster)](https://build.incquerylabs.com/jenkins/job/MagicDraw-OSS/job/plugin-skeleton/job/master/)
+
 ## How to use:
 
 - Install product: Eclipse IDE for Eclipse Committers, version: Photon
-- Install VIATRA SDK 2.0.1 http://download.eclipse.org/viatra/updates/release/latest
-- Install UML 2.5.1 metamodel: 
+- Install VIATRA SDK 2.0.1 or newer http://download.eclipse.org/viatra/updates/release/latest
+- Install UML 2.5.1 metamodel or newer: 
```

Comment:
This is not or newer - the metamodel version is included in the nsURI, so it will not work if other versions would be used.

Reply:
Ohh, so that's an exact version. I got it.

Reply:
But what will happen if a newer target platform refers a different version of the metamodel?

Reply:
At that point, we have to rewrite all the queries in the repository, so we will detect it, and at that point we will have to update this readme file as well.

In case of VIATRA version, we aim for API compatibility between generator versions: compiling with an older version should still work in a newer runtime, and we regularly update the runtimes so the users should be fine.